docs/references/v2.openapi.json

```
@@ -808,6 +808,342 @@
+          },
+          "401": {
+            "$ref": "#/components/responses/401"
+          },
+          "404": {
+            "$ref": "#/components/responses/404schema"
+          }
+        },
+        "parameters": [
+          {
+            "schema": {
+              "type": "string"
+            },
+            "name": "id",
+            "in": "path",
+            "required": true,
+            "description": "`id`of the schema"
+          }
+        ]
+      },
+      "put": {
+        "summary": "Update schema",
+        "operationId": "put-update-new-schema",
+        "description": "Updates the body for the schema. A schema can only be updated if it is not used by any room.",
+        "requestBody": {
+          "content": {
+            "application/json": {
+              "schema": {
+                "$ref": "#/components/schemas/UpdateSchema"
+              },
+              "examples": {
+                "example-1": {
+                  "value": {
+                    "body": "type Logo {\nname: string\ntheme: string\n}\n\ntype Storage {\nlogo: LiveObject<Logo>\n"
+                  }
+                }
+              }
+            }
+          }
+        },
+        "responses": {
+          "200": {
+            "description": "Success. Updates the new schema and increments the version. The schema `body` is returned as JSON.",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/Schema"
+                },
+                "examples": {
+                  "example-1": {
+                    "value": {
+                      "id": "my-schema@2",
+                      "name": "my-schema",
+                      "version": 2,
+                      "createdAt": "2023-03-30T07:25:54.675Z",
+                      "updatedAt": "2023-03-30T07:25:54.675Z",
+                      "body": "type Logo {\nname: string\ntheme: string\n}\n\ntype Storage {\nlogo: LiveObject<Logo>\n"
+                    }
+                  }
+                }
+              }
+            }
+          },
+          "401": {
+            "$ref": "#/components/responses/401"
+          },
+          "403": {
+            "$ref": "#/components/responses/403schema"
+          },
+          "422": {
+            "$ref": "#/components/responses/422schema"
+          }
+        },
+        "parameters": [
+          {
+            "schema": {
+              "type": "string"
+            },
+            "name": "id",
+            "in": "path",
+            "required": true,
+            "description": "`id`of the schema"
+          }
+        ]
+      },
+      "delete": {
+        "summary": "Delete schema",
+        "operationId": "delete-a-schema",
+        "description": "This endpoint deletes a schema by its id. The id is a combination of the schema name and version. For example, `my-schema@1`. A schema can only be deleted if it is not attached to a room.",
```

We haven't been consistent with this in the past, but I think "id" should be spelled "ID" everywhere. Not just here. Possibly good to do a pass over all documentation and fix all such cases in a single change.

```suggestion
        "description": "This endpoint deletes a schema by its ID. The ID is a combination of the schema name and version. For example, `my-schema@1`. A schema can only be deleted if it is not attached to a room.",
```